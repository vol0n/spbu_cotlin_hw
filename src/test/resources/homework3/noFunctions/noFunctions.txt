package noFunctions

internal class NoFunctionsTest
