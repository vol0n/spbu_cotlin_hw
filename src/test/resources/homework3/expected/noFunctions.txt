package noFunctions

public class NoFunctionsTest
